Library/Homebrew/shims/super/xz

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+XZ="$(/usr/bin/which "$HOMEBREW_PREFIX/opt/xz/bin/xz" ||
+      echo /usr/bin/xz)"
```

Comment:
I'd just always use the `HOMEBREW_PREFIX/opt/xz/bin/xz` and not create a shim given this path doesn't exist on macOS.